Source/WebCore/mathml/MathMLElement.cpp

```diff
@@ -268,8 +268,12 @@ bool MathMLElement::isKeyboardFocusable(KeyboardEvent* event) const
     if (isFocusable() && StyledElement::supportsFocus())
         return StyledElement::isKeyboardFocusable(event);
 
-    if (isLink())
-        return document().frame()->eventHandler().tabsToLinks(event);
+    if (isLink()) {
+        RefPtr frame = document().frame();
+        if (!frame)
+            return false;
+        return frame->eventHandler().tabsToLinks(event);
```

Comment:
Optional: this is a slightly nicer pattern to use here:
```suggestion
        return frame && frame->eventHandler().tabsToLinks(event);
```